CoreFoundation/Collections.subproj/CFData.c

```diff
@@ -322,19 +324,16 @@ static const CFRuntimeClass __CFDataClass = {
 };
 
 CFTypeID CFDataGetTypeID(void) {
-    static dispatch_once_t initOnce;
-    dispatch_once(&initOnce, ^{ __kCFDataTypeID = _CFRuntimeRegisterClass(&__CFDataClass); });
-    return __kCFDataTypeID;
+    return _kCFRuntimeIDCFData;
 }
 
 void _CFDataInit(CFMutableDataRef memory, CFOptionFlags variety, CFIndex capacity, const uint8_t *bytes, CFIndex length, Boolean noCopy) {
     Boolean isMutable = ((variety & __kCFMutableMask) != 0);
     Boolean isGrowable = ((variety & __kCFGrowableMask) != 0);
-    Boolean isDontDeallocate = ((variety & __kCFDontDeallocate) != 0);
     
     __CFDataSetNumBytesUsed(memory, 0);
     __CFDataSetLength(memory, 0);
-    __CFDataSetDontDeallocate(memory, isDontDeallocate);
+    __CFDataSetDontDeallocate(memory, true);
```

Comment:
I think this is reverting an earlier change

Reply:
Yup. To fix.